core/src/test/java/org/jdbi/v3/core/TestHandle.java

```diff
@@ -183,7 +183,7 @@ public void testCommitRollback() throws Exception {
     public void testIssue2065() throws Exception {
         h.begin();
         h.execute("insert into something (id, name) values (1, 'Brian')");
-        String value = h.createQuery("select name from something where id = 1").mapToBean(Something.class).one().getName();
+        final String value = h.createQuery("select name from something where id = 1").mapToBean(Something.class).one().getName();
```

Comment:
why final?

Reply:
`VariableDeclarationUsageDistance` again. `final` variables are not checked.